ddp-workspace/projects/ddp-dsm-ui/src/app/participant-page/participant-page.component.ts

```diff
@@ -652,8 +652,7 @@ export class ParticipantPageComponent implements OnInit, OnDestroy, AfterViewChe
         v = value.checked;
       }
     }
-    if (v !== null) {
-
+    if (v !== null || parameterName.includes('faxSent')) {
```

Comment:
Since we have `faxSent` hardcoded in a few places, it might be helpful to pull this out to a constant.  And since this check will apply to faxSent, faxSent2, and faxSent3, it might be better to check those values explicitly, on the off chance that we make a `faxSent` variant that shouldn't trigger this logic.

Reply:
Or a separate method like `isFaxParameter` that can take in a parameter name and return whether or not it's a parameter that is subject to this special handling.